Tools/setup/docker-entrypoint.sh

```diff
@@ -1,19 +1,27 @@
 #!/bin/bash
 
+GREEN='\033[0;32m'
+NC='\033[0m' # No Color
+FILE_DESCRIPTOR="${GREEN}[docker-entrypoint.sh]${NC}"
```

Comment:
File [descritor](https://en.wikipedia.org/wiki/File_descriptor) is a pretty specific thing. https://en.wikipedia.org/wiki/File_descriptor

Reply:
TIL! thanks for pointing this out, I updated to another name